pdi-step-plugin-archetype/src/main/resources/archetype-resources/src/main/java/__plugin_class_name__.java

```diff
@@ -35,16 +35,20 @@
  * Describe your step plugin.
  * 
  */
-public class ${plugin_class_name} extends BaseStep implements StepInterface {
-	
-	private static Class<?> PKG = ${plugin_class_name}Meta.class; // for i18n purposes, needed by Translator2!!   $NON-NLS-1$
-	
-	public ${plugin_class_name}(StepMeta stepMeta, StepDataInterface stepDataInterface, int copyNr, TransMeta transMeta,
-		Trans trans) {
-		super(stepMeta, stepDataInterface, copyNr, transMeta, trans);
-	}
-	
-	/**
+public class $ {
+  plugin_class_name}extends BaseStep implements StepInterface{
+
+  private static Class<?> PKG = ${plugin_class_name}Meta.class; // for i18n purposes, needed by Translator2!! $NON-NLS-1$
+
+  public ${plugin_class_name}(
+  StepMeta stepMeta, StepDataInterface stepDataInterface,
+  int copyNr, TransMeta transMeta,
+  Trans trans)
+  {
```

Comment:
probably would fail checkstyle. starting curly braces should not be on their own line. there is also no indentation of the parameters on new lines